### PR TITLE
Correct spelling errors

### PR DESCRIPTION
**Pull request by banan314**

### CODE REVIEW

**Comment by banan314** [comment]
Only the last commit (f36b212) is important, the first 3 are merges.

File: tests/actions.py [hunk 1]
```diff
@@ -12,15 +12,15 @@
 class TestPrimaryKeyBinding(tests.TestCase):
 
 	def runTest(self):
-		for accel in ( # Ctrl-A or Command-A
```
Comment (banan314) [inline tests/actions.py]: I know it's a shortage, but in my opinion `accelerator` is better.

File: zim/config/dicts.py [hunk 2]
```diff
@@ -689,7 +689,7 @@ def setdefault(self, key, default, check=None, allow_empty=False):
 
 class SectionedConfigDict(ControlledDict):
 	'''Dict with multiple sections of config values
-	Sections are auto-vivicated when a non-existing item is retrieved.
```
Comment (banan314) [inline zim/config/dicts.py]: there is no such a word

File: zim/config/dicts.py [hunk 3]
```diff
@@ -880,8 +880,8 @@ class HierarchicDict(object):
 	There is a special member dict stored under the key "__defaults__"
 	which has the top-level fallback properties.
 
-	Child dicts are auto-vivicated, so this object only implements
```
Comment (banan314) [inline zim/config/dicts.py]: there is no such a word

**Comment by jaap-karssenberg** [comment]
Looks like this closed automatically when I dropped the "next" branch -- could you re-open the request for "master" ?

Thanks,

Jaap

**Pull request closed without merging**